go-runtime/compile/schema_test.go

```diff
@@ -262,35 +269,37 @@ func TestErrorReporting(t *testing.T) {
-			filename+":36:2-46: call must have exactly three arguments\n"+
-			filename+":37:16-25: call first argument must be a function\n"+
-			filename+":42:1-2: must have at most two parameters (context.Context, struct)\n"+
-			filename+":42:69-69: unsupported response type \"ftl/failing.Response\"\n"+
-			filename+":47:22-27: first parameter must be of type context.Context but is ftl/failing.Request\n"+
-			filename+":47:37-43: second parameter must be a struct but is string\n"+
-			filename+":47:53-53: unsupported response type \"ftl/failing.Response\"\n"+
-			filename+":52:43-47: second parameter must not be ftl.Unit\n"+
-			filename+":52:59-59: unsupported response type \"ftl/failing.Response\"\n"+
-			filename+":57:1-2: first parameter must be context.Context\n"+
-			filename+":57:18-18: unsupported response type \"ftl/failing.Response\"\n"+
-			filename+":62:1-2: must have at most two results (struct, error)\n"+
-			filename+":62:41-44: unsupported request type \"ftl/failing.Request\"\n"+
-			filename+":67:1-2: must at least return an error\n"+
-			filename+":67:36-39: unsupported request type \"ftl/failing.Request\"\n"+
-			filename+":71:35-38: unsupported request type \"ftl/failing.Request\"\n"+
-			filename+":71:48-48: must return an error but is ftl/failing.Response\n"+
-			filename+":76:41-44: unsupported request type \"ftl/failing.Request\"\n"+
-			filename+":76:55-55: first result must be a struct but is string\n"+
-			filename+":76:63-63: must return an error but is string\n"+
-			filename+":76:63-63: second result must not be ftl.Unit\n"+
-			filename+":83:1-1: duplicate verb name \"WrongResponse\"\n"+
-			filename+":89:2-12: struct field unexported must be exported by starting with an uppercase letter",
+			filename+":29:2-23: cannot attach enum value to BadValueEnum because it is a variant of type enum TypeEnum, not a value enum\n"+
+			filename+":35:2-40: cannot attach enum value to BadValueEnumOrderDoesntMatter because it is a variant of type enum TypeEnum, not a value enum\n"+
+			filename+":41:2-10: unsupported type \"error\" for field \"BadParam\"\n"+
+			filename+":44:2-17: unsupported type \"uint64\" for field \"AnotherBadParam\"\n"+
+			filename+":47:3-3: unexpected token \"export\" (expected Directive)\n"+
+			filename+":53:36-39: unsupported request type \"ftl/failing.Request\"\n"+
+			filename+":53:50-50: unsupported response type \"ftl/failing.Response\"\n"+
+			filename+":54:16-29: call first argument must be a function in an ftl module\n"+
+			filename+":55:2-46: call must have exactly three arguments\n"+
+			filename+":56:16-25: call first argument must be a function\n"+
+			filename+":61:1-2: must have at most two parameters (context.Context, struct)\n"+
+			filename+":61:69-69: unsupported response type \"ftl/failing.Response\"\n"+
+			filename+":66:22-27: first parameter must be of type context.Context but is ftl/failing.Request\n"+
+			filename+":66:37-43: second parameter must be a struct but is string\n"+
```

Comment:
We should try to start putting new entries at the bottom of the input to this test, so that the line numbers don't shift on every change.

Reply:
Awesome, done.